src/command/exec.ts

```diff
@@ -0,0 +1,49 @@
+import { Wallet } from "ethers"
+import { CommandModule } from "yargs"
+import { formatInfo, formatProperty, formatTitle } from "../util/format"
+import { fetchConfiguration, fetchMetadata, Contracts } from "../util/metadata"
+
+import { getProvider } from "../util/provider"
+import { getStageName } from "../util/stage"
+
+import { Actions, layerOfFunction, actionOfFunction, actionMaps } from "../util/functionsMap"
+import { PERP_MNEMONIC } from "../util/constant"
+import yaml from "js-yaml"
+import fs from "fs"
+import path from "path"
+
+const execCommand: CommandModule = {
+    command: "exec <file_name>",
+    describe: "interact with contracts",
+    builder: yargs =>
+        yargs.positional("file_name", {
+            describe: "a yml format file",
+            type: "string",
+        }),
+
+    handler: async argv => {
+        const stageName = getStageName(argv.stage)
+        const metadata = await fetchMetadata(stageName)
+        const config = await fetchConfiguration(stageName)
+        const file = argv.file_name as string
+        console.log(formatInfo(`** send the following txs on ${stageName} **\n`))
+
+        const fullExecFilePath = path.join(__dirname, file)
```

Comment:
should use `path.resolve()` here. `__dirname` means the directory which contains the javascript file. e.g. if we compile typescript to javascript, it will be in `<rootdir>/build/src/command/`, it would fail if we execute `perp execute <file>` since the file is not in `<rootdir>/build/src/command/`.

if use `path.resolve()` it will consider the relative path and solve it to an absolute path.